tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/metrics/CircuitBreakerMetricTest.java

```diff
@@ -122,9 +123,17 @@ public void testCircuitBreakerMetric() throws Exception {
         assertThat("circuitbreaker calls failed", m.getCircuitBreakerCallsFailedDelta(), is(2L));
         assertThat("circuitbreaker calls prevented", m.getCircuitBreakerCallsPreventedDelta(), is(1L));
         assertThat("circuit breaker times opened", m.getCircuitBreakerOpenedDelta(), is(1L));
-        
+
+        // exception that is considered a success
+        expect(RuntimeException.class, () -> cbBean.doWork(Result.PASS_EXCEPTION));
+
+        assertThat("circuitbreaker calls succeeded", m.getCircuitBreakerCallsSucceededDelta(), is(3L));
+        assertThat("circuitbreaker calls failed", m.getCircuitBreakerCallsFailedDelta(), is(2L));
+        assertThat("circuitbreaker calls prevented", m.getCircuitBreakerCallsPreventedDelta(), is(1L));
+        assertThat("circuit breaker times opened", m.getCircuitBreakerOpenedDelta(), is(1L));
+
         // General metrics should be updated
-        assertThat("invocations", m.getInvocationsDelta(), is(5L));
+        assertThat("invocations", m.getInvocationsDelta(), is(6L));
         assertThat("failed invocations", m.getInvocationsFailedDelta(), is(3L));
```

Comment:
I'd want to run it to check, but I think this counter should also be incremented.

This is the counter for `ft.<name>.invocations.failed.total`, defined as "The number of times the method was called and, after all Fault Tolerance actions had been processed, threw a Throwable"

Reply:
Ah right! Didn't notice that, I was still thinking in terms of what circuit breaker considers failure. Will fix.

Reply:
Fixed.